WooCommerce/Classes/Extensions/UIImage+Woo.swift

```diff
@@ -237,6 +237,12 @@ extension UIImage {
         UIImage.gridicon(.house)
     }
 
+    /// House Outlined Image
+    ///
+    static var houseOutlinedImage: UIImage {
+        UIImage(named: "icon-house-outlined")!
```

Comment:
nit: you can use `UIImage(imageLiteralResourceName: "woo-no-store")` which doesn't force you to use the `!` although it will probably crash in the same way if the resource doesn't exist! 🤷

Reply:
ohh I do prefer not having `!` in the code! updated in https://github.com/woocommerce/woocommerce-ios/pull/3398/commits/7196589609c05d515ae512d5ea827f836890e2f1